src/format.ts

```diff
@@ -45,15 +45,6 @@ interface Format {
     display: Display;
 }
 
-
 // ----- Exports ----- //
 
-export {
-    Pillar,
-    Special,
-    Theme,
-    Design,
-    Display,
-};
-
-export type { Format };
+export type { Pillar, Special, Theme, Design, Display, Format };
```

Comment:
Is it ok to export `enum`s as `type`s? I may be wrong, but I thought this had caused us some problems previously? See [this discussion](https://github.com/guardian/apps-rendering/pull/888#discussion_r517262592).

Or was that just because they were _imported_ as types in AR, but not _exported_ as types in `@guardian/types`? 🤔

Reply:
I'm really not sure. TS was telling me to add them. I can only add them to ophan, which is my current problem, but it seems like we should standardise?

I wonder if `@guardian/eslint-config-typescript` would enforce it actually?

Reply:
I pulled your branch into AR to test it. Unfortunately we run into this error for all the enums:

```
TS1362: 'Design' cannot be used as a value because it was exported using 'export type'.
```

Essentially whenever there's a line of code that looks like this:

```ts
    if (design === Design.Comment) {
```

TypeScript doesn't know what to do, because it needs `Design.Comment` to be a value so it can be used in runtime code.

Reply:
ok – that's pretty convincing! will change that